Makefile

```diff
@@ -639,7 +638,6 @@ SPEED_OPTIMISED_SRC := $(SPEED_OPTIMISED_SRC) \
             drivers/rx_nrf24l01.c \
             drivers/rx_spi.c \
             drivers/rx_xn297.c \
-            drivers/pwm_output.c \
```

Comment:
@ledvinap: https://gcc.gnu.org/onlinedocs/gcc/Common-Function-Attributes.html, the section for `optimize` (coupled with the remark on what the `GCC optimize` pragma does on https://gcc.gnu.org/onlinedocs/gcc/Function-Specific-Option-Pragmas.html